modules/google-analytics/wp-google-analytics.php

```diff
@@ -50,7 +50,9 @@ class Jetpack_Google_Analytics {
 	 * @return void
 	 */
 	private function __construct() {
-		if ( Jetpack_Google_Analytics_Options::enhanced_ecommerce_tracking_is_enabled() ) {
+		// At this time, we only leverage universal analytics when enhanced ecommerce is selected and WooCommerce is active.
+		// Otherwise, don't bother emitting the tracking ID or fetching analytics.js
```

Comment:
Maybe reword or even completely remove this comment because it is a little confusing since the else condition will emit a tracking ID ala legacy.